src/loader.c

```diff
@@ -14,6 +15,335 @@ uint8_t mcb_alloc_st = 0;
 // PSP (Program Segment Prefix) location
 uint16_t current_PSP;
 
+//#define FCB_DOS11
```

Comment:
Instead of this, please do:
```
/* Selects FCB parsing behavior, must be 1, 2 or 3: */
#define FCB_PARSE_DOS 3
```
This allows simplifying the tests bellow.

Reply:
Done